docker-compose.yml

```
@@ -30,6 +30,8 @@
        - postgres:/data/postgres
     networks:
       - postgres
+    mem_limit: 200M
+    mem_reservation: 50M  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)